### DISCUSSION TITLE
Is it possible to monitoring every channel keda is probing?

### DISCUSSION THREAD

**Discussion by oak-tree** [kedacore/keda#1539] [Q&A / Need Help]
Hey,

Because all workers can go to zero scale. We do not know if they are working on in failure state. Is it possible for Keda to report "probing the following providers"

"Azure service bus: channel X, subsription Y"
"Redis list X"
...

**Comment by tomkerkhove** [comment]
We are adding Kubernetes events for this https://github.com/kedacore/keda/pull/1523, would that help?

**Reply by zroubalik** [reply]
and ScaledObject's Status could give you some additional info as well (if you don't want to check KEDA Operator logs)

**Reply by oak-tree** [reply]
Interesting, I would also add a `listening` event every `x`, even if the length of queue/subscription is zero. This means that  if there is zero message for long time, we can still monitor that `keda` works and listen to everything it should.
